### DISCUSSION TITLE
Missing github_token file for github_api_showcase.Rmd

### DISCUSSION THREAD

**Discussion by anthonyjlau** [sailuh/kaiaulu#267] [Q&A] [answered]
In github_api_showcase.Rmd, it expects a file called github_token that stores the Github Token API. The file does not appear in the file directory. Are we supposed to make this file or download it from somewhere?

**Comment by Ssunoo2** [comment, marked as answer]
Hi Anthony, I ran into this problem as well! There are instructions to creating a github token here: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token). For the scope, I did not check any boxes, though I'm not sure if that part was correct. It will generate you a string to use. In the specified directory, create a plain text file and paste the code inside and this should solve the problem (eg. touch github_token, vim github_token). Note that the .ssh file may be present but hidden in your home directory.

**Reply by carlosparadis** [reply]
Yes, this is accurate. The Notebook should explain you need the token created. I am curious if the confusion here was that the file related to mention or unfamiliarity with the use of tokens in API in general. Let me know if you can get it to work!

**Reply by anthonyjlau** [reply]
I was able to generate my Github token and put it into the .ssh folder. I am curious to know more about Github tokens and how they are used. Thanks!

**Reply by carlosparadis** [reply]
@anthonyjlau Thank you for confirming it worked in the end. What caused the confusion? Did the notebook not make clear how to create the github_token file?

**Reply by anthonyjlau** [reply]
I was able to create my token with the instructions but I was unsure of where to put it. I also did not know what scope to use. I did not know about the hidden folders so I did not know where the .ssh folder was. I also did not know I needed to create the github_token file myself.

**Reply by carlosparadis** [reply]
Thank you so much!
